payments-core/src/main/java/com/stripe/android/model/parsers/ElementsSessionJsonParser.kt

```diff
@@ -43,6 +43,9 @@ internal class ElementsSessionJsonParser(
         val linkFlags = json.optJSONObject(FIELD_LINK_SETTINGS)?.let { linkSettingsJson ->
             parseLinkFlags(linkSettingsJson)
         } ?: emptyMap()
+        val useLinkRebrand = json.optJSONObject(FIELD_FLAGS)
+            ?.optBoolean(FIELD_LINK_REBRAND, ElementsSession.LinkSettings.useRebrandDefault)
+            ?: ElementsSession.LinkSettings.useRebrandDefault
```

Comment:
Ah I see, it passed as a general flag. I'm surprised this isn't part of `LinkSettings` flags.

Reply:
+1 I asked the link team about changing it!